src/main/java/racingcar/step3/RacingCar.java

```diff
@@ -23,11 +32,57 @@ public int getCountOfRounds(){
         return countOfRounds;
     }
 
-    public int[] getCars(){
-        return cars;
+    public ArrayList<Car> getInfoOfCars(){
+        return infoOfCars;
     }
 
-    public ArrayList getMovementOfCar(){
+    public ArrayList<Integer> getMovementOfCar(){
         return movementOfCar;
     }
+
+    public ArrayList<String> getWinner(){
+        return winner;
+    }
+
+    public void race(RacingCar racingCar) {
+        for (int i=0; i < countOfCars; i++){
+            racingCar.move(i);
+            movementOfCar.add(getInfoOfCars().get(i).getPositionOfCar());
+        }
+    }
+
+    public void move(int i) {
+        if (RandomNumber.makeRandomNumber() >= MOVE_CONDITION) {
+            getInfoOfCars().get(i).moveCar();
+        }
+    }
+
+    public ArrayList<Car> enterNameOfCars(String nameOfCars){
+        String[] namesOfCar = nameOfCars.split(",");
+        infoOfCars = new ArrayList<>();
+        for (int i=0; i < namesOfCar.length; i++){
+            infoOfCars.add(new Car(namesOfCar[i], 0));
+        }
```

Comment:
```suggestion
        for (String carName : namesOfCar) {
            infoOfCars.add(new Car(carName, 0));
        }
```
이렇게 변경하면 조금더 심플한거 같아요😁